### ISSUE TITLE
Use project 

### ISSUE THREAD

**Issue by fif10** [ymajoros/winbooks-java#6]
Hello 

I would like to use your library 
But when I run the tests, I get the exeption below. 
The version I have currently installed is the demo version. (6.50.360) 
Do I need to run any special command to register dll or any other requiremts ? 

(anyway, thanks a lot. I am looking forward to use your lib  !!! ) 

com4j.ExecutionException: com4j.ComException: 80040154 CoCreateInstance failed : Classe non enregistrée : .\com4j.cpp:153

	at com4j.ComThread.execute(ComThread.java:203)
	at com4j.Task.execute(Task.java:25)
	at com4j.COM4J.createInstance(COM4J.java:97)
	at com4j.COM4J.createInstance(COM4J.java:72)
	at be.valuya.winbooks.ClassFactory.createWinbooksObject(ClassFactory.java:125)
	at be.valuya.jbooks.Winbooks.<init>(Winbooks.java:92)
	at be.valuya.jbooks.WinbooksTest.setUp(WinbooksTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
Caused by: com4j.ComException: 80040154 CoCreateInstance failed : Classe non enregistrée : .\com4j.cpp:153
	at com4j.Native.createInstance(Native Method)
	at com4j.COM4J$CreateInstanceTask.call(COM4J.java:117)
	at com4j.COM4J$CreateInstanceTask.call(COM4J.java:104)
	at com4j.Task.invoke(Task.java:51)
	at com4j.ComThread.run0(ComThread.java:153)
	at com4j.ComThread.run(ComThread.java:134)

Process finished with exit code -1

Thanks

**Comment by cghislai** [comment]
Try running the mvn install goal: you are missing some dependencies
(defined in winbooks-api-client/pom.xml):

<dependency>
    <groupId>org.jvnet.com4j</groupId>
    <artifactId>com4j</artifactId>
    <version>20120426-2</version>
</dependency>

**Comment by fif10** [comment]
Thank for the quick reply ! 
I had indead some dependencies missing (jdbf) 
I managed to run the maven install goal 

I had to skip tests because api-extra need some data, but even setting my baseFolderPath to PARFILUX data, tests are failing afterward : 
java.lang.IllegalArgumentException: Unreadable docOrder: ESC
	at be.valuya.jbooks.model.WbDocOrderType.getDocOrderTypeNumber(WbDocOrderType.java:40)
	at be.valuya.jbooks.model.WbDocOrderType.lambda$fromString$1(WbDocOrderType.java:33)

But managed to build successfully, but still have the same error in api-client...

**Comment by fif10** [comment]
I almost became crazy. But found the issue :-)

I looked a bit in com4j and tried to understand it. Anyway, I found that the issue was due of  the 64bits
So I played by registrering / deregistering librairies (com4j and winbooks) : no luck 
And I finally succeded by changing my java jdk version to 32bits

Then I had a "dossier not found", so I deregistrered the WinbooksOfficeApi.dll from the SysWow64 windows folder : 
`regsvr32.exe \u c:\winbooks\WinbooksOfficeApi.dll`
moved it to my winbooks install folder and re-registred it : 
`regsvr32.exe c:\winbooks\WinbooksOfficeApi.dll`

Now almost all the tests are ok.
Thanks a lot for your work !!!
